Libraries/PeriphDrivers/Include/MAX32690/wut.h

```diff
@@ -198,36 +204,36 @@ int MXC_WUT_GetTicks(uint32_t time, mxc_wut_unit_t units, uint32_t *ticks);
  * @return     #E_NO_ERROR If everything is successful. 
  * @return     @ref MXC_Error_Codes If function is unsuccessful.
  */
-int MXC_WUT_GetTime(uint32_t ticks, uint32_t *time, mxc_wut_unit_t *units);
+int MXC_WUT_GetTime(mxc_wut_regs_t *wut, uint32_t ticks, uint32_t *time, mxc_wut_unit_t *units);
 
 /**
  * @brief   Wait for an edge of the WUT count register.
  */
-void MXC_WUT_Edge(void);
```

Comment:
Potential alternative name: `MXC_WUT_WaitForEdge()`

Reply:
I think that's a good idea. It's a better self-descriptive function name than just `Edge`